### PR TITLE
Added more tests to the Keyboard and App.js, hit 70% coverage

### PR DESCRIPTION
**Pull request by brunogrcsada**
### Current code coverage:

image

We need to hit at least 90% to 100%. Currently, a lot more testing needs to be done on the Keyboard component (both manual and through the React Testing Library)

### CODE REVIEW

**Review by flynnn231 — APPROVED** [review]

File: src/components/keyboard/keyboard.test.js [hunk 1]
```diff
@@ -1,6 +1,40 @@
 import { render, screen } from '@testing-library/react';
 import Keyboard from './keyboard';
 
-it('should...', () => {
-
+describe('Test that keyboard keys are rendered correctly', () => {
```
Comment (Jsimpada) [inline src/components/keyboard/keyboard.test.js:4]: Great set of tests, but we need to add a few more Jest tests for logic.

File: src/components/volume/volume.test.js [hunk 2]
```diff
@@ -1,6 +1,40 @@
...
+        expect(screen.getByTestId('sliderItem').min).toBe('1');
+        expect(screen.getByTestId('sliderItem').max).toBe('100');
+    });
+
+    it('should have a default value from the inherited prop', () => {
+        render(<Volume volume={50}/>);
+        expect(screen.getByTestId('sliderItem').value).toBe('50');
+    });
+    
+    it('should move the slider if input value is changed', () => {
```
Comment (Jsimpada) [inline src/components/volume/volume.test.js:32]: Nice workaround, it's very difficult to test audio :(

**Review by Jsimpada — APPROVED** [review]
Awesome work Bruno, keep it up :1st_place_medal:

**Review by melissalupu — APPROVED** [review]
Looks good! Let's try and push the code coverage to 90%

**Review by elijaholan — APPROVED** [review]
Nice! We've met our coverage criteria, but I agree that we should bump it up further.

**Pull request merged**